doc/release-notes-24198.md

```diff
@@ -0,0 +1,5 @@
+Updated RPCs
+------------
+
+- The following RPCs: `listtransactions`, `gettransaction` and `listsinceblock`
+  now include the `wtxid` of the transaction.
```

Comment:
nit: Probably a better way to phrase this

Reply:
Hm, interesting. Maybe:

"Every transaction returned by the following RPCs: `listtransactions`, `gettransaction` and `listsinceblock` now include a `wtxid` field (hash of serialized transaction, including witness data)."

Reply:
Suggest "The `listtransactions`, `gettransaction`, and `listsinceblock` RPC methods now include a `wtxid` field (hash of serialized transaction, including witness data) for each transaction."